integrations/docker/images/chip-cert-bins/README.md

```diff
@@ -0,0 +1,46 @@
+# Docker image for Matter Certification Test Harness
+
+The Dockerfile here helps build multi-platform Docker images containing the  executable binaries necessary for the Matter Test Harness. It utilizes the BuildKit toolkit and Buildx included within Docker since version 18.06.
+
+## Running
+
+In order to properly run the binaries, avahi must be properly set up and passed to the container.
+
+Prerequisites:
+
+* Host must support and enable IPv6 and be on a network that has IPv6.
+* IPv6 must be enabled within avahi config on the host. ```use-ipv6=yes``` in avahi-daemon.conf
```

Comment:
should we also have `use-ipv4=no` as a prerequisite? 
Should this be done on the docker file as well?

Reply:
use-ipv4=yes is OK and does not seem to have an impact, I wouldn't change this as it may impact the user's other software which depends on avahi. This isn't in a Docker because it's the host side's configuration.